frontend/tailwind.config.js

```diff
@@ -0,0 +1,35 @@
+import { theme } from '@ifixit/ui/theme';
+
+const { colors, breakpoints, space, radii, shadows, fonts, zIndices } = theme;
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+   content: ['./app/**/*.{js,ts,jsx,tsx,mdx}'],
+   theme: {
+      colors,
+      screens: breakpoints,
+      spacing: space,
+      borderRadius: applyTailwindDefault(radii, ['md', 'base']),
+      boxShadow: applyTailwindDefault(shadows, 'md'),
+      // dropShadow: applyTailwindDefault(shadows, 'md'),
+      extend: {
+         fontFamily: fonts,
+         zIndex: zIndices,
+      },
+   },
+   plugins: [],
+};
+
+function applyTailwindDefault(themePropertyMap, defaultKeys) {
+   const keySet = new Set(
+      Array.isArray(defaultKeys) ? defaultKeys : [defaultKeys]
+   );
+   return Object.keys(themePropertyMap).reduce((acc, key) => {
+      if (keySet.has(key)) {
+         acc['DEFAULT'] = themePropertyMap[key];
+      } else {
+         acc[key] = themePropertyMap[key];
+      }
+      return acc;
+   }, {});
```

Comment:
It is correct.
To be the most adherent possible to the `react-commerce` theme defaults, I inherited the styles from @ifixit/ui/theme. However, I noticed that in our theme `borderRadius` included more than one key with the same associated value:

```tsx
export const radii: ThemeOverride['radii'] = {
   sm: primitiveBorderRadius.sm,
   md: primitiveBorderRadius.md,
   lg: primitiveBorderRadius.lg,
   xl: primitiveBorderRadius.xl,
   base: primitiveBorderRadius.md,
   full: primitiveBorderRadius.pill,
};
```

To remove noise I merged `md` and `base` keys under `DEFAULT`.

Reply:
Nevermind, I had to remove this borderRadius customization because a missing `rounded-md` class could potentially break some tailwindUI component. So I also removed the array of `defaultKeys` transforming it into a single `defaultKey` string